storage/tests/system.py

```diff
@@ -242,6 +242,61 @@ def test_bucket_update_labels(self):
         bucket.update()
         self.assertEqual(bucket.labels, {})
 
+    def test_get_set_iam_policy(self):
+        import pytest
+        from google.cloud.storage.iam import STORAGE_OBJECT_VIEWER_ROLE
+        from google.api_core.exceptions import BadRequest, PreconditionFailed
+        from google.api_core.iam import InvalidOperationException
+
+        bucket_name = "iam-policy" + unique_resource_id("-")
+        bucket = retry_429_503(Config.CLIENT.create_bucket)(bucket_name)
+        self.case_buckets_to_delete.append(bucket_name)
+        self.assertTrue(bucket.exists())
+
+        policy_no_version = bucket.get_iam_policy()
+        self.assertEqual(policy_no_version.version, 1)
+
+        policy = bucket.get_iam_policy(requested_policy_version=3)
+        self.assertEqual(policy, policy_no_version)
+
+        member = "serviceAccount:{}".format(Config.CLIENT.get_service_account_email())
+
+        BINDING_W_CONDITION = {
+            "role": STORAGE_OBJECT_VIEWER_ROLE,
+            "members": {member},
```

Comment:
Should this be a list []?

Reply:
We made it so that the `members` field in bindings is a set: https://github.com/googleapis/google-cloud-python/pull/9869/files#diff-b0f82e58dc32268923d71a91e4270829R176